### ISSUE TITLE
Can't withdraw coins from Ledger Live

### ISSUE THREAD

**Issue by so-sweet** [LedgerHQ/ledger-live-desktop#1781]
Everytime I try to withdraw from the Ledger Live - I get this message:
Wrong device or passphrase for ‘Stratis 1’
Please use the device or passphrase for the selected account

It doesn't matter what coin I try to withdraw - it gives me the same message with that particular coin and I can't withdraw.  Can someone help me with this.  I don't know what is going on.
I have the most recent update for Ledger Live and the Nano S, so it should be working, but it isn't. Very frustrating!

**Comment by gre** [comment]
Can you attempt a clear cache to see if it helps?

we suspect inconsistencies between `derivationMode` and `freshAddressPath`.

(1)

can you go to your Account and open its Settings (the wrench icon next to Receive) and then in the modal click on Advanced Logs.

Can you tell me what you have just in **freshAddressPath** ? (do not share anything else, don't share here your xpub)

(2)

After that, can you attempt to do a Add Accounts again from stratis and see if Add Accounts modal will tell you your account is already imported or if it sees it as new account?

(3)

can you attempt to do a clear cache, does it solve the problem?

(4)

can you attempt to remove the account from LL desktop and add it again?

**Comment by so-sweet** [comment]
Yes - I cleared the cache.  My freshAddressPath says:  "44'/144'/0'/0/0"
I am trying to add the Stratis again and it says go to your Stratis on your
device and it just sits there going around in a circle.  But when I clicked
on the Stratis on the Device, this time it did not give me that message.
But when I went to the XRP and entered the address and Tag number, I got
that error message again.  Very frustrating. So I can't withdraw the XRP.
I cleared the cache before I hit Send and went through the process.  Here's
what I noticed on the API though - the node says 2Ripple, but the
description in my accounts says Ripple1.
Clearing cache does not solve the problem
Guess what - I removed both accounts Strat and xrp and now it shows zero in
my Ledger Live - what is going on?

**Comment by Alotofwhoknowswhat** [comment]
I decided to load the 1.3.2 Ledger Live because I downloaded the 1.3.4 before and see if I can see my coins there, but it kept telling me to verifiy to get my coins, so I verified the Receive addresses, but it still showed zero coins in all my accounts.  I just want to withdraw my coins.  Sigh

**Comment by Alotofwhoknowswhat** [comment]
Maybe someone from Ledger can send me a link to a new Ledger Live that works?

**Comment by Alotofwhoknowswhat** [comment]
Ledger Live is still showing ZERO on my coin amounts.   How do I get them to show again? Hope this can get fixed ASAP!

**Comment by meriadec** [comment]
Not sure I get your issue, do you confirm that if you search your address on explorer (e.g for [stratis](https://chainz.cryptoid.info/strat/)) you see your balance, but you don't see the same in Ledger Live? If yes, you may open a support ticket, providing your xpub.

**Comment by so-sweet** [comment]
Yes. I am saying that the amounts have been showing in Ledger Live every
time I went in there and  now they are not there

**Comment by so-sweet** [comment]
Okay. Thanks for your help. I did contact support but no reply so far

**Comment by so-sweet** [comment]
Can you get Ledger to answer back to my emails?

Peggy
